pkg/virt-api/webhooks/validating-webhook/admitters/vmi-create-admitter.go

```diff
@@ -2027,6 +2028,43 @@ func getNumberOfPodInterfaces(spec *v1.VirtualMachineInstanceSpec) int {
 	return nPodInterfaces
 }
 
+func validateClientDevices(field *k8sfield.Path, clientDevices []v1.ClientDevice) []metav1.StatusCause {
+	var causes []metav1.StatusCause
+
+	if len(clientDevices) == 0 {
+		// That's fine as this is optional
+		return nil
+	} else if len(clientDevices) < 0 {
```

Comment:
How can this happen?

Reply:
This build in function will never return `<0`.

Reply:
Removed this chunk in the last rebase.
Thanks!